properly register conjugate/neg fallthroughs to prim ops

bdhirsh opened the pull request:
A few aten ops (like `clone` and `copy_` get fallthrough registrations to the Conjugate/Negative keys. We haven't been giving the same treatment to their corresponding `prims` variants, which can cause infinite loops in some cases.

Fixes an infinite loop that showed up in tests from https://github.com/pytorch/pytorch/pull/132563


Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #132699

bdhirsh committed:
properly register conjugate/neg fallthroughs to prim ops

[ghstack-poisoned]

albanD reviewed (approved):
I would have comments for the API but it's internal so ok!

bdhirsh committed:
Update on "properly register conjugate/neg fallthroughs to prim ops"

A few aten ops (like `clone` and `copy_` get fallthrough registrations to the Conjugate/Negative keys. We haven't been giving the same treatment to their corresponding `prims` variants, which can cause infinite loops in some cases.




[ghstack-poisoned]

bdhirsh commented:
@pytorchbot merge
